src/content/docs/integrations/prometheus-integrations/install-configure/remote-write-drop-data.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Drop data using Prometheus remote write 
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure remote write
+translate:
+  - jp
+---
+
+You can drop data you aren't interested in keeping by changing the `remote_write` section of the YAML config file.
```

Comment:
Line 11: I had to re-read "You can't drop data you aren't interested in keeping by changing..." a couple of times in order to understand what it was trying to say. This could be a nitpick.

I would suggest rewriting the first part of this sentence so there isn't two `-ing` words so close together. EX) `You can drop data you don't want by changing` or similar.

Reply:
Fixed, thanks!